chromadb/config.py

```diff
@@ -74,6 +75,9 @@
     "chromadb.segment.distributed.MemberlistProvider": "chroma_memberlist_provider_impl",
 }
 
+DEFAULT_TENANT = "default"
+DEFAULT_DATABASE = "default"
```

Comment:
can we make these `default_tenant` and `default_database` respectively? I can see this being confusing during debugging / logging.